frontend/src/components/filter/Filter.tsx

```diff
@@ -217,31 +229,21 @@ const Filter: React.FC = (props: any) => {
         </div>
       ));
   }
-  // /  for showing more
-  const seeMore = (e: any) => {
-    const x = max;
-
-    if (x < status.length) {
-      setMax(status.length);
-      setShow('Show less');
-    } else {
-      setMax((x) => x - 13);
-      setShow('Show all');
-    }
-  };
 
   return (
-    <div className="filter-size" >
+    <div className="filter-size" key="">
```

Comment:
why is the key empty? wouldn't this create issues? doesn't react expect keys to be unique?

Reply:
here by mistake I have used and need to remove it. react expect keys to be unique.